jans-cedarling/cedarling/src/jwt/test/with_validation/access_token.rs

```diff
@@ -250,15 +255,18 @@ fn errors_on_invalid_signature() {
         "exp": Timestamp::one_hour_after_now(),
     });
 
-    // generate the signed access_token
-    let access_token = generate_token_using_claims(&access_token_claims, &encoding_keys[0]);
+    // generate the access_token with invalid signature
+    let access_token = generate_token_using_claims(&access_token_claims, &encoding_keys[0])
+        .unwrap_or_else(|e| panic!("Failed to generate access_token: {:?}", e));
```

Comment:
why do not use `.expect("message")`?

Reply:
fixed here [1d282ce](https://github.com/JanssenProject/jans/pull/9982/commits/1d282ce6e2077f9ee5768e2a51d42c347285a380)